### PR TITLE
Attach question pool to cohort

### PR DESCRIPTION
**Pull request by timothymarotta**
need to review overwriteCohortDatastoreFileTest in CohortRecordTest. Are there any more edge cases you can see that I've missed?

### CODE REVIEW

File: src/test/java/edu/ithaca/dragon/par/io/CohortRecordTest.java [hunk 1]
```diff
@@ -152,27 +166,110 @@ public void cohortRecordsToFromJsonTest() throws IOException {
...
+    @Test
+    public void overwriteCohortDatastoreFileTest() throws IOException {
+        JsonIoUtil jsonIoUtil = new JsonIoUtil(new JsonIoHelperDefault());
+        //read in 2 cohort datastores- 1 not altered for comparison, 1 to alter
+        List<CohortRecord> testCohortRecords = jsonIoUtil.listFromFile("src/test/resources/author/defaultCohortDatastore.json", CohortRecord.class);
+        JSONCohortDatastore toReference = CohortRecord.makeCohortDatastoreFromCohortRecords(testCohortRecords, "src/test/resources/author/defaultCohortDatastore.json");
+        JSONCohortDatastore toTest = CohortRecord.makeCohortDatastoreFromCohortRecords(testCohortRecords, "src/test/resources/author/defaultCohortDatastore.json");
+
+        //ask for TaskGenerator of studentID that does not exist
+        TaskGenerator taskGeneratorRequested = toTest.getTaskGeneratorFromStudentID("taskGeneratorStudent");
```
Comment (tobydragon) [inline src/test/java/edu/ithaca/dragon/par/io/CohortRecordTest.java:224]: would proabably be good to check that it wasn't there before calling to ensure you're testing the right thing.

**Review by tobydragon — COMMENTED** [review]
Otherwise, looks great. Good testing work
